docs/unlocked_package.rst

```diff
@@ -1,14 +1,65 @@
-Release an unlocked package
+Release an Unlocked Package
 ===========================
 
+While CumulusCI was originally created to develop managed packages, it can also be used to develop and release `unlocked packages <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_intro.htm>`_.
+
+
+
 Prerequisites
 -------------
 
-Create a package version
+To create unlocked package versions, first complete these steps.
+
+* `Enable DevHub Features in Your Org <https://developer.salesforce.com/docs/atlas.en-us.packagingGuide.meta/packagingGuide/sfdx_setup_enable_devhub.htm>`_.
+* `Enable Unlocked and Second-Generation Managed Packaging <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_setup_enable_secondgen_pkg.htm>`_.
+* Connect the DevHub org to the CumulusCI keychain by running ``cci org connect devhub``. (This is necessary even if ``sfdx`` has already authenticated to the DevHub.)
+* To create an unlocked package with a namespace, you must also create a new Developer Edition org to `Create and Register Your Namespace <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_create_namespace.htm>`_, and link the namespace to your DevHub.
+
+
+
+Create a Package Version
 ------------------------
 
-Promote a package version
+To create a new unlocked package version, run the ``create_package_version`` task against a scratch org.
+
+.. code-block:: console
+
+    $ cci task run create_package_version --org <org_name> --package_type Unlocked
+
+This task looks in your default DevHub org (as configured in ``sfdx``) for an unlocked package with the name and namespace specified in the task options (defaulting to the name and namespace from the ``project__package`` section of the ``cumulusci.yml`` file). If a matching package doesn't exist, the task then submits a request to create the package version. When completed (which can take some time), the task outputs a ``SubscriberPackageVersion Id``, which can be used to install the package in another org.
```

Comment:
Anyone know what the standard is for "id" in the style guide? In my experience, it's usually ID or id, but in these docs it was "Id".

Reply:
I have a habit of capitalizing only the 'I' from how [SOQL queries](https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql.htm) were written in the Salesforce documentation. I'm not aware of a standard, but we could always address something like this if it comes up later.

Reply:
Changing it to ID. Will confirm with Liane next week.